boot/seal.go

```diff
@@ -406,35 +406,35 @@ func resealKeyToModeenvSecboot(rootdir string, model *asserts.Model, modeenv *Mo
 		return fmt.Errorf("cannot compose run mode boot chains: %v", err)
 	}
 
-	// reseal the run object
-	pbc := toPredictableBootChains(append(runModeBootChains, recoveryBootChainsForRunKey...))
-
-	needed, nextCount, err := isResealNeeded(pbc, bootChainsFileUnder(rootdir), expectReseal)
```

Comment:
I just noticed we have typo in the doc comment of isResealNeeded:  s/rease count/reseal count/